src/generate-ova-from-template.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+
+import struct
+import tempfile
+import os
+import sys
+import json
+from subprocess import check_call
+from stat import *
+from shutil import rmtree, move
+import tarfile
+
+# Usage: generate-ova-from-template.py input_image.qcow2 input_template.xml example_ova_parameters.json output_image.ova
+
+def qcow_disk_size(filename):
+    # Detect if an image is in qcow format
```

Comment:
This is great documentation!! Do you mind updating it to be the docstring?

IE:

```python
    """
    Detect if an ....
    <snip />
    """
     # And in Python struct format string-ese
    qcow_struct=">IIQIIQIIQQIIQ" # > means big-endian
```